addons/website/views/website_navbar_templates.xml

```diff
@@ -20,8 +20,15 @@
             <t t-set="body_classname" t-value="(body_classname if body_classname else '') + (' o_connected_user' if env['ir.ui.view'].user_has_groups('base.group_user') else '')"/>
         </xpath>
         <xpath expr="//div[@id='wrapwrap']" position="before">
-            <div t-if="'with_loader' in request.params" class="o_theme_install_loader_container position-fixed fixed-top fixed-left h-100 w-100 d-flex justify-content-center align-items-center">
+            <div t-if="'with_loader' in request.params" class="o_theme_install_loader_container position-fixed fixed-top fixed-left
```

Comment:
make this code as a template and call it in both places where it is used... I'm pretty sure that the 7 will become a 6 in a few week, and we will forget that this code is duplicated.

call wrapped with a  <t t-if> here

Reply:
didn't see that one qweb js and on qweb py

-> solved